packages/kbn-es/src/utils/docker.test.ts

```diff
@@ -103,6 +110,32 @@ describe('resolveDockerImage()', () => {
   });
 });
 
+describe('resolvePort()', () => {
+  test('should return default port when no options', () => {
+    const port = resolvePort({});
+
+    expect(port).toMatchInlineSnapshot(`
```

Comment:
This was more convenience to get a quick test in and possibly update in the future. It may be required to update ports on all nodes instead of just master node in the future.